src/kaleidoscope/driver/mcu/AVR.h

```diff
@@ -0,0 +1,77 @@
+/* -*- mode: c++ -*-
+ * driver::MCU::AVR -- AVR MCU driver for Kaleidoscope
```

Comment:
Is this actually going to be true for every AVR? Or should we name this class ATMega32U4?

Reply:
I'm not sure. This should work for the ATMega32U4 and a few others, but perhaps not all AVRs. I guess the safest route is to rename it to ATMega32U4, and if we start supporting other AVR MCUs, we can figure something out then.